papf_node:
  ros__parameters:

    lookahead_distance: 10.0
    voxel_size: 0.1
    obstacle_point_radius: 0.1
    virtual_wall_points: 10
    virtual_wall_distance: 0.5

    max_plan_steps: 500
    goal_tolerance: 1.0

    papf.k_att: 5.0
    papf.d_g: 3.0

    papf.k_rep: 2.0 
    papf.d_o: 0.5
    papf.n: 2

    papf.k_prd: 2.0
    papf.d_prd: 2.0

    papf.max_turning_angle_deg: 2.0
    papf.v_c: 2.0  
    papf.v_min: 0.5
    papf.v_max: 3.0 
    papf.max_acceleration: 1.0
    papf.max_deceleration: 1.0
    papf.dt: 0.05